### PR TITLE
Pass info from FEEDS to ItemExporter

### PR DESCRIPTION
**Pull request by maranqz**
Resolves https://github.com/scrapy/scrapy/issues/4606

### CODE REVIEW

**Comment by maranqz** [comment]
The second commit was fail. In third one I do nothing and tests passed. I think it is not correct.

File: docs/topics/feed-exports.rst [hunk 1]
```diff
@@ -332,6 +335,8 @@ as a fallback value if that key is not provided for a specific feed definition:
 
 -   ``indent``: falls back to :setting:`FEED_EXPORT_INDENT`.
 
+-   ``item_export_kwargs``: dict with kwargs for :ref:`Item exporters <topics-exporters>` classes.
```
Comment (Gallaecio) [inline docs/topics/feed-exports.rst:338]: :lipstick: 

```suggestion
-   ``item_export_kwargs``: :class:`dict` with keyword arguments for the corresponding :ref:`item exporter class <topics-exporters>`.
```

**Review by Gallaecio — APPROVED** [review]
> In third one I do nothing and tests passed. I think it is not correct.

I don’t know what you mean. If I comment out the feedexport.py line, the new test fails as expected.

As far as I can tell, good work!

**Comment by maranqz** [comment]
> > In third one I do nothing and tests passed. I think it is not correct.
> 
> I don’t know what you mean. If I comment out the feedexport.py line, the new test fails as expected.
> 
> As far as I can tell, good work!

I mean, there is problem with tests.
Testing of commit https://github.com/scrapy/scrapy/pull/4768/commits/fc3c66ce950e945e7ac6e19fea31e8454147ac29 was fail. Then I create https://github.com/scrapy/scrapy/pull/4768/commits/71d2c2f1a319b570eb2026707d49c4f62d59296e without valuable changes and tests passed.

**Comment by Gallaecio** [comment]
> I mean, there is problem with tests.
> Testing of commit [fc3c66c](https://github.com/scrapy/scrapy/commit/fc3c66ce950e945e7ac6e19fea31e8454147ac29) was fail. Then I create [71d2c2f](https://github.com/scrapy/scrapy/commit/71d2c2f1a319b570eb2026707d49c4f62d59296e) without valuable changes and tests passed.

Oh, I see! I thought you meant there was a problem in your code, not in the CI. My bad!

Looks like it was a case of random failure. We have a few of those related to the logging of warnings and errors, sometimes they pick unrelated logging messages causing a test to fail. Nothing for you to worry about.

**Comment by kmike** [comment]
Thanks @maranqz!

**Pull request merged**